### DISCUSSION TITLE
"old" problem for a new Windows 10 user

### DISCUSSION THREAD

**Discussion by NBcobo** [LostRuins/koboldcpp#151] [Q&A] [answered]
I just installed Windows 10 for the sake of these LLM applications. When I drag and drop a ggml .bin file to koboldcpp.exe, I got similar errors like what I had with Windows 7. Not sure how to fix this:
![image](https://user-images.githubusercontent.com/130814579/236706107-52c30e71-13e0-4733-9016-f3d0f7362519.png)
I do have Python and C++ installed.

**Comment by LostRuins** [comment]
Do you have enough RAM to run this model? How much RAM do you have? Can you test with a smaller model first?

https://huggingface.co/concedo/cerebras-111M-ggml/blob/main/cerberas-111m-q4_0.bin

**Reply by NBcobo** [reply]
I have 64GB of RAM, which should be enough. Since I have a newly installed Windows 10 (N LTSC 2021), what is the best way to install any prerequisite, like Python or C++? Any caveat? I read somewhere that Anaconda may not be a good choice for Python.

**Reply by LostRuins** [reply]
There are no prerequisites required on windows. Everything runs from the pyinstaller exe that is self contained and contains all dependencies.

**Reply by NBcobo** [reply]
The error message is identical to part of this old error message which I cannot replicate somehow. Seems I may have some issue with is koboldcpp_openblas.dll file?
![image](https://user-images.githubusercontent.com/130814579/236969260-0fda3069-e316-4c01-ab1c-77460ab00933.png)

**Reply by LostRuins** [reply]
Yes, it does seem like the dll file doesn't exist for you for some reason. Here are some things you can try:
1. You're on v1.18. Try the latest version (v1.20) here: https://github.com/LostRuins/koboldcpp/releases/tag/v1.20 
2. If you have python installed, you can run the koboldcpp.py file directly instead of using the exe. That way you can ensure that all the necessary dll files are present. You can download the zip here: https://github.com/LostRuins/koboldcpp/releases/download/v1.20/koboldcpp.zip 
3. If the dlls do not appear, you may have some sort of antivirus or antimalware software that is flagging and removing files as a false positive. You may want to whitelist the dlls or temporarily disable it.

**Reply by NBcobo** [reply, marked as answer]
Thank you!
I found a closed issue https://github.com/LostRuins/koboldcpp/issues/56 which is exactly like mine.
It turns out my CPU only supports AVX, not AVX2. With the --noavx2 option, I can get the chat going finally.
Great appreciate all your help!
